spacebench/api/dataverse.py

```diff
@@ -91,80 +69,73 @@ def list_data_files(self, include_fileid=False):
                 result.append(f"{file_name}\t{file_desc}\t{file_id}")
             else:
                 result.append(f"{file_name}\t{file_desc}")
-        
+
         return "\n".join(result)
 
     def remove_temp_files(self):
-        """ Removes temporary files. """
+        """Removes temporary files."""
 
         for filename in self.files:
-            file_path = os.path.join(
-                self.temp_dir, filename
-            )
+            file_path = os.path.join(self.dir, filename)
             if os.path.isfile(file_path):
                 os.remove(file_path)
-                LOGGER.info(
-                    f"{filename} removed from the temporary directory.")
+                LOGGER.info(f"{filename} removed from the temporary directory.")
 
-    def download_data(self, predictor, type):
-        """ Downloads core data and dicts from Dataverse. """
-    
-        self.__get_filenames_from_arguments(
-            predictor, type)
-        self.__get_fileids_from_filenames()
+    def download_data(self, name: str) -> str:
+        """Downloads core data and dicts from Dataverse."""
 
-        for filename in self.files:
-            filename_temp_path = os.path.join(
-                self.temp_dir, filename
-            )
+        # Download core data
+        fileid = self.files[name]
+        filename_temp_path = os.path.join(self.dir, name)
 
-            if os.path.exists(filename_temp_path):
-                LOGGER.info(
-                    f"File {filename} already exists in the temporary directory.")
-            else:
-                response = self.data_api.get_datafile(
-                    self.files[filename])
+        if os.path.exists(filename_temp_path):
+            LOGGER.info(f"File {name} already exists in the temporary directory.")
+        else:
+            response = self.data_api.get_datafile(fileid)
+
+            with open(filename_temp_path, mode="wb") as temp_file:
+                temp_file.write(response.content)
+
+            LOGGER.info(
+                f"Downloaded: filename {name}, id {fileid}, saved to {filename_temp_path}"
+            )
 
-                with open(filename_temp_path, mode="wb") as temp_file:
-                    temp_file.write(response.content)
-                
-                LOGGER.info(
-                    "Downloaded: file name {}, id {}".format(
-                    filename, self.files[filename]))
+        return filename_temp_path
 
     def publish_dataset(self, token):
         """
-        Publish new dataset. 
+        Publish new dataset.
 
         Args:
             token (str): Dataverse API Token.
         """
         api = NativeApi(self.base_url, token)
         resp = api.publish_dataset(self.pid, release_type="major")
         if resp.json()["status"] == "OK":
-           LOGGER.info("Dataset published.")
+            LOGGER.info("Dataset published.")
```

Comment:
The log file could be more informative. This particular line, along with several others, could be enhanced with additional information. Please consider including the name of the dataset for better clarity and usefulness.